src/commands/createFunctionApp/stacks/getStackPicks.ts

```diff
@@ -72,6 +81,15 @@ export async function getStackPicks(context: IFunctionAppWizardContext): Promise
     }
 
     return picks.sort((p1, p2) => {
+        if (hasEndOfLife) {
+            (picks as IAzureQuickPickItem<FullFunctionAppStack | undefined>[]).push({
+                label: localize('endOfLife', `$(extensions-warning-message) Some stacks have an end of support deadline coming up. Learn more...`),
+                onPicked: async () => {
+                    await openUrl('https://aka.ms/AA1tpij');
+                },
+                data: undefined
+            });
+        }
```

Comment:
Doing this here will result in many of these picks being pushed into `picks`. Remove the return here, and push the pick in after sorting, then `return picks`.